### DISCUSSION TITLE
Should we be using these in decision making?

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#219] [Polls]
Github added polling to the discussions feature. 

I'm wondering if this would be useful to use with making decisions on changes to this site?

**Comment by duboisp** [comment]
The pool feature seems to be a great feature but I have some concern about how we can get our people to use the GitHub Discussion web interface. For a developer that is perfect, but a little bit harder for our content writer and communication folks.
